specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -259,6 +309,20 @@ model CertificateMetadata {
   name?: string;
 }
 
+@added(APIVersions.v2023_04_01_preview)
+@doc("Auto stop criteria for a test.")
+model AutoStopCriteria {
+  @doc("Auto stop is enabled or not")
+  autoStopEnabled?: boolean;
```

Comment:
This is slight misleading as if the customer provides a criteria it already means it is enabled isnt ?

Reply:
I think the behavior is we can provide AutoStopCriteria and then update it disable or enable. So the test can still have AutoStopCriteria but can be disabled. Let me confirm this with feature owner.